src/Main.java

```diff
@@ -1,5 +1,49 @@
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
 public class Main {
     public static void main(String[] args) {
+        //задача1
+        int[] salary = {1500, 2000, 500, 3000, 4000};
+        int spending = 0;
+        for (int element : salary) {
+            spending = spending + element;
+        }
+        System.out.println("Сумма трат за месяц составила " + spending + " рублей");
+        // задача 2
+        int[] salary1 = {1500, 2000, 500, 3000, 4000};
+        int minSpending = salary1[0];
+        int maxSpending = salary1[0];
+        for (int element : salary1) {
+            if (element < minSpending) {
+                minSpending = element;
+            }
+            if (element > maxSpending) {
+                maxSpending = element;
+            }
+        }
+        System.out.println("Минимальная сумма трат за неделю составила " + minSpending + " рублей. Максимальная сумма трат за неделю составила " + maxSpending + " рублей");
+        // задача 3
+        int[] salary2 = {1500, 2000, 500, 3000, 4000};
+        int spending1 = 0;
+        int numberWeeks = 5;
```

Comment:
Не нужно хардкодить значения, можно количество недель взять из массива
```suggestion
        int numberWeeks = salary2.length;
```